src/index.js

```diff
@@ -0,0 +1,57 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const addBtn = document.getElementById('add');
+const list = document.getElementById('list');
+
+class Book {
+  Constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const titleText = title.value;
+  const authorText = author.value;
+  const newBook = new Book(titleText, authorText);
+  if (localStorage.getItem('books')) {
+    let savedBooks = JSON.parse(localStorage.getItem('books'));
+    savedBooks = [newBook, ...savedBooks];
+    localStorage.setItem('books', JSON.stringify(savedBooks));
+  } else {
+    localStorage.setItem('books', JSON.stringify([newBook]));
+  }
+  document.location.reload();
+});
+
+const listbooks = JSON.parse(localStorage.getItem('books'));
+
+const removeTheBook = (id) => {
+  const list = listbooks.filter((book) => book.title !== listbooks[id].title);
+  localStorage.setItem('books', JSON.stringify(list));
+  document.location.reload();
+};
```

Comment:
- [ ] When saving the data to the local storage, it seems the application is saving an empty object. There seems to be a problem with this function. When retrieving the data since the object is empty then it returns undefined and this is what is displayed on the page. Kindly double-check the saving of the information to local storage

Reply:
@peterrobert  Thanks! 
It was a misspelled constructor keyword.